ips/src/tests.rs

```diff
@@ -200,12 +219,73 @@ fn destroy_should_fail() {
         assert_eq!(
             IpsStorage::<Runtime>::get(0),
             Some(IpsInfoOf::<Runtime> {
-                owner: primitives::utils::multi_account_id::<Runtime, <Runtime as Config>::IpsId>(
-                    0
-                ),
+                parentage: Parentage::Parent(primitives::utils::multi_account_id::<
+                    Runtime,
+                    <Runtime as Config>::IpsId,
+                >(0, None)),
                 metadata: MOCK_METADATA.to_vec().try_into().unwrap(),
-                data: vec![0]
+                data: vec![AnyId::IpfId(0)].try_into().unwrap(),
+                allow_replica: true,
+                ips_type: IpsType::Normal,
             })
         );
     });
 }
+
+#[test]
+fn create_replica_should_work() {
```

Comment:
This unit test is missing the case where Bob creates an ips with allow_replica set to true and Alice then creates a replica, add this alongside the already implemented case.

Reply:
Resolved on 340697b